### PR TITLE
Upgrade to php8

### PR DESCRIPTION
**Pull request by masterix21**

### CODE REVIEW

File: src/Landlord.php [hunk 1]
```diff
@@ -10,11 +10,11 @@ public static function execute(callable $callable)
...
 
         Tenant::forgetCurrent();
-        
+
         $result = $callable();
-        
-        optional($originalCurrentTenant)->makeCurrent();
-        
+
+        $originalCurrentTenant?->makeCurrent();
```
Comment (freekmurze) [inline src/Landlord.php:16]: Yes! 🙂

**Comment by freekmurze** [comment]
Could you use the `PackageServiceProvider` from https://github.com/spatie/laravel-package-tools ?

**Comment by masterix21** [comment]
Sure, I will.

**Comment by masterix21** [comment]
@freekmurze, please verify now. 

Thanks

**Comment by freekmurze** [comment]
Very nice thanks!

**Pull request merged**